flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -155,10 +157,19 @@ public class YarnClusterDescriptor implements ClusterDescriptor<ApplicationId> {
     /** True if the descriptor must not shut down the YarnClient. */
     private final boolean sharedYarnClient;
 
-    /** Lazily initialized list of files to ship. */
-    private final List<File> shipFiles = new LinkedList<>();
+    /**
+     * Lazily initialized list of files to ship. The path string for the files which is configured
+     * by {@code YarnConfigOptions#SHIP_FILES} will be converted to {@code Path} with schema and
```

Comment:
```suggestion
     * by {@link YarnConfigOptions#SHIP_FILES} will be converted to {@link Path} with schema and
```

It should be link instead of code, the link can be clicked in IDEA. 

The `shipArchives` is same.